libr/core/cmd.c

```diff
@@ -1322,27 +1322,6 @@ static int cmd_pipein(void *user, const char *input) {
 	return 0;
 }
 
-static void task_test(RCore *core, int usecs) {
```

Comment:
since you are removing this, please also remove the `#if 0` code in `core/cmd.c` where task_test is "used".